pr-data.csv

```diff
@@ -992,6 +992,19 @@ https://github.com/apache/cxf,649520354501295859b692ddf1b42adb919affbb,tools/wad
 https://github.com/apache/cxf,649520354501295859b692ddf1b42adb919affbb,rt/transports/http-netty/netty-server,org.apache.cxf.transport.http.netty.server.integration.NettyServerTest.testGetWsdl,ID,,,
 https://github.com/apache/cxf,649520354501295859b692ddf1b42adb919affbb,rt/transports/jms,org.apache.cxf.transport.jms.RequestResponseTest.testRequestTopicResponseStaticQueue,ID,,,
 https://github.com/apache/dolphinscheduler,26a84b30d822b59925a82fa6724df24fd5c13213,dolphinscheduler-spi,org.apache.dolphinscheduler.spi.params.PluginParamsTransferTest.testGetParamsJson,ID,Opened,https://github.com/apache/dolphinscheduler/pull/6773,
+https://github.com/apache/druid,50963edcae70150f13520b619f167512d951a71b,core,org.apache.druid.java.util.emitter.core.EmitterTest.testBatchSplitting,ID,,,
```

Comment:
For these detected tests, do you have any logs of NonDex run as proof?

Reply:
Yes, they are in my vm under the `druid` folder's `.nondex`. Should I provide them in some way?

Reply:
> Yes, they are in my vm under the `druid` folder's `.nondex`. Should I provide them in some way?

We want the command line output for running NonDex to show that these tests are flaky. The `.nondex` folder is hard to read.

Reply:
ok, I'm going to work on that!

Reply:
@xieyt2000 Hi Yuntong, I've temporarily removed other changes. Could you please check these changes to see whether they can be accepted? Appreciate that. :)